docs/reference/UDS Core/prerequisites.md

```diff
@@ -136,3 +140,180 @@ Metrics server is provided as an optional component in UDS Core and can be enabl
   optionalComponents:
     - metrics-server
 ```
+
+#### Loki
+
+The Loki deployment is (by default) backed by an object storage provider for log retention.  For cloud environments you can wire this into the environment's storage provider with the following overrides:
+
+```yaml
+loki:
+  values:
+    - path: loki.storage.s3.endpoint
+      value: "<s3-endpoint>"
+    - path: loki.storage.s3.secretAccessKey
+      value: "<s3-secret-key>"
+    - path: loki.storage.s3.accessKeyId
+      value: "<s3-access-key>"
+    - path: loki.storage.bucketNames.chunks
+      value: "<chunks-bucket-name>"
+    - path: loki.storage.bucketNames.ruler
+      value: "<ruler-bucket-name>"
+    - path: loki.storage.bucketNames.admin
+      value: "<admin-bucket-name>"
+    - path: loki.storage.bucketNames.region
+      value: "<s3-region>"
+```
+
+You can also use the [Minio Operator UDS Package](https://github.com/defenseunicorns/uds-package-minio-operator) to back Loki with the following overrides:
+
+```yaml
+- name: minio-operator
+  ...
+  overrides:
+    minio-operator:
+      uds-minio-config:
+        values:
+          - path: apps
+            value:
+              - name: loki
+                namespace: loki
+                remoteSelector:
+                  app.kubernetes.io/name: loki
+                bucketNames:
+                  - uds-loki-chunks
+                  - uds-loki-ruler
+                  - uds-loki-admin
+                copyPassword:
+                  enabled: true
+
+- name: core-logging
+  ...
+  overrides:
+    loki:
```

Comment:
We should also add the overrides to lock down the egress traffic to strictly what is necessary here (`uds-loki-config` chart has some [values for storage egress](https://github.com/defenseunicorns/uds-core/blob/add-docs-s3-lb-prerequisites/src/loki/chart/values.yaml) that when not specified default to egress anywhere).